src/lib/mina_base/snapp_predicate.ml

```diff
@@ -281,6 +269,16 @@ module Eq_data = struct
         ; equal_checked = run equal_var
         }
 
+    let token_id =
+      Token_id.
+        { default
+        ; to_input_checked = Checked.to_input
+        ; to_input
+        ; typ
+        ; equal
+        ; equal_checked = Checked.equal
```

Comment:
I think the field in `src/app/archive/snapp_tables.sql` will need updating for this (sorry for missing at the last review..)